docs/get_started.rst

```diff
@@ -1,17 +1,341 @@
+
+These commands can take several minutes to complete.
+Once finished, you can `verify your installation`_.
+
+
+
+On Windows
+^^^^^^^^^^
+
+
+
+Install Python 3
+********************
+1. Go to the `Python downloads page <https://www.python.org/downloads/windows/>`_.
+2. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
+3. Use the installation wizard to install.
+   *Be sure to check the "Add Python to environment variables" checkbox at the end of the install wizard*,
+   otherwise you may encounter a ``command not found`` error with the next step.
+   To access the Advanced Options area, select "Customize Installation" then click through Optional features page.
+
+.. image:: images/windows_python.png
+
+
+
+Install via ``pipx``
+***********************
+
+Open your preferred terminal application
+(e.g. `CMD.exe <https://www.bleepingcomputer.com/tutorials/windows-command-prompt-introduction/>`_ on Windows).
+If you already have your terminal open, close it and reopen it. Enter the following command::
+
+    $ python -m pip install --user pipx
+
+.. image:: images/pipx.png
+
+To permanently modify the default environment variables, follow these steps. Note that to change System variables, you need non-restricted access to your machine (i.e. Administrator rights).
+
+1. Click Start and search for â€˜edit environment variablesâ€™ or open System properties,
+   Advanced system settings.
+2. Click the Environment Variables button.
+3. To change System variables, you need non-restricted access to your machine
+   (i.e. Administrator rights). Add the following paths to your PATH environment variable:
+
+   a. ``%USERPROFILE%\AppData\Roaming\Python\Python37\Scripts``
+   b. ``%USERPROFILE%\.local\bin``
+
+.. image:: images/env-var.png
+
+Open a new command prompt and verify that pipx is available::
+
+    $ pipx --version
+
+You should see a version number after entering in this command, such as: ``0.12.3.1``.
+If you get an error instead, such as ``'pipx' is not recognized as an internal or external command,
+operable program or batch file.``, please check that your environment variables have been updated.
+
+You can now install CumulusCI with::
+
+    $ pipx install cumulusci
+
+Now `verify your installation`_.
+
+
+
+Verify Your Installation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a new terminal window can verify that CumulusCI is installed correctly by running ``cci version``:
+
+.. code:: console
+
+    $ cci version
+    CumulusCI version: 3.19.0 (/path/to/bin/cci)
+    Python version: 3.8.5 (/path/to/bin/python)
+
+    You have the latest version of CumulusCI.
+
+You can also use this command in the future to check whether your CumulusCI installation is up to date.
+
+Still need help? `CumulusCI's issues on GitHub <https://github.com/SFDO-Tooling/CumulusCI/issues>`_ may have something useful.
+
+
```

Comment:
Good question. There's a short bit in Scratch Orgs that covers it briefly, but it might fit better here. Putting it here might align better with Trailhead given that we do all of the setup of a piece there. I'm not sure I have a very strong opinion, though.

Reply:
@davisagli - I'm going to include part of the section we have currently in the tutorial on setting up sfdx.